CMakeLists.txt

```diff
@@ -13,8 +13,8 @@ project(FunGame VERSION 0.1.0)
 # add the executable
 file(GLOB_RECURSE SOURCES1 "src/*.cpp")
 
+# this is just one json file
 file(GLOB SOURCES4 "vendor/*.c*")
-file(GLOB SOURCES5 "vendor/json/*.c*")
```

Comment:
how does this compile???

Reply:
I don't know. Is the answer add executable?

Reply:
i mean i think you should keep this line